template/java8/gradlew

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
```

Comment:
Do these files need to be duplicated in both locations?

Reply:
There already are `wrapper` folders in all of the modules, but they won't work without the corresponding files. I suppose a better solution might be to delete those. Does that sound better?

Reply:
I was about to write the same issue.
It would be handy to have the gradle wrapper in the handler module.
If it's not needed I would remove the wrapper folders from the other modules.

Reply:
@alexellis what do you think?